Eastern-Research-Group/mywaterway
# Feature/update usgs streamgages display

## Related Issues:
* https://jira.epa.gov/browse/HMW-4

## Main Changes:
Further refinements of USGS streamgages on the Community Overview page:
* Created a parameter mapping file for mapping returned USGS STA parameter data with formatted parameter name and units, and prioritized parameter order for display in the HMW app.
* Parameter codes are now displayed after the parameter name in parenthesis in the accordion/listview body.
* Updated the streamgages to all be displayed in a single color. Code for conditionally rendering gage color, based on defined color stops has been commented out for now, so if the USGS STA updates their web service to include this data, we can easily introduce the color coding.
* Begin to solve the issue of duplicate parameters being returned (although the current solution isn't comprehensive enough – I'll demonstrate the flaws in our next meeting with EPA).

## Steps To Test:
1. Navigate to http://localhost:3000/community/020700100102/overview
2. Switch to the "Monitoring & Sensors" tab, toggle the "Daily Water Conditions" switch, and expand the first item in the listview below
3. Honestly, there's nothing really to visually test/verify, as the details of the parameters (and the nuances of their naming and ordering) will be discussed with EPA tomorrow.
